### DISCUSSION TITLE
Pages number

### DISCUSSION THREAD

**Discussion by ghost** [koreader/koreader#11646] [Q&A]
Is there any way to show the page number close to a real book? **Like in PocketBook (see photo attachment)** software for example. Not 1000+ but 400. I hope you understand me. It will be very helpful and necessary 🤩
I will show you examples in photos ![IMG_20240409_135116.jpg](https://github.com/koreader/koreader/assets/119916135/2eddeee8-53d5-416e-a6ba-895507bc8d32)

![IMG_20240409_135225.jpg](https://github.com/koreader/koreader/assets/119916135/c871d05c-c53a-4429-a6d2-465c35bf9006)

**Comment by jonnyl2** [comment]
See issue #9020. There is a user patch available there, too. I haven't tried it yet though, so I can't comment on it. (Applies if the book doesn't already come with publisher-defined reference page numbers, but that's relatively rare and I think they're enabled by default when available.)

**Reply by ghost** [reply]
> See issue #9020. There is a user patch available there, too. I haven't tried it yet though, so I can't comment on it. (Applies if the book doesn't already come with publisher-defined reference page numbers, but that's relatively rare and I think they're enabled by default when available.)

I downloaded the patch (.lua file), how do I need to install this? Can you explain please

**Reply by jonnyl2** [reply]
Just copy it into the /patches subfolder of your KOReader directory (create if it doesn't exist yet). Make sure it's appropriately named (`2-... .lua`).

More info here: https://github.com/koreader/koreader/wiki/User-patches

Upon starting KOReader, it will automatically be activated. You can manually deactivate it under `Tools -> More tools -> Patch management ->`

**Reply by ghost** [reply]
> Just copy it into the /patches subfolder of your KOReader directory (create if it doesn't exist yet). Make sure it's appropriately named (`2-... .lua`).
> 
> More info here: https://github.com/koreader/koreader/wiki/User-patches
> 
> Upon starting KOReader, it will automatically be activated. You can manually deactivate it under `Tools -> More tools -> Patch management ->`
> 

Thank you very very much 😊 
I adore KOReader 🙏
